pyacq/core/stream/sharedmemstream.py

```diff
@@ -9,7 +9,25 @@
 from .ringbuffer import RingBuffer
 
 class SharedMemSender(DataSender):
-    """
+    """Stream sender that uses shared memory for efficient interprocess
+    communication. Only the data pointer is sent over the socket.
+    
+    Note: this class is usually not instantiated directly; use
+    ``OutputStream.configure(transfermode='sharedmem')``.
+    
+    Extra parameters accepted when configuring the output stream:
+    
+    * shape (tuple) The shape of one frame (a sample at one time point).
```

Comment:
For multi channel, we can put that shape=(-1, nb_channel) is a good practice.

This avoid to mix up with buffer_size.

Reply:
Ah, good catch. I removed shape from the sender-specific docs, since the OutputStream docs already cover it correctly.